responsibleai/tests/test_rai_insights_validations.py

```diff
@@ -264,13 +265,52 @@ def test_mismatch_train_test_features(self):
         assert 'The features in train and test data do not match' in \
             str(ucve.value)
 
+    def test_dirty_train_test_data(self):
+        X_train = pd.DataFrame(data=[['1', np.nan], ['2', '3']],
+                               columns=['c1', 'c2'])
+        y_train = np.array([1, 0])
+        X_test = pd.DataFrame(data=[['1', '2'], ['2', '3']],
+                              columns=['c1', 'c2'])
+        y_test = np.array([1, 0])
+
+        model = LGBMClassifier(boosting_type='gbdt', learning_rate=0.1,
```

Comment:
nit: use "create_lightgbm_classifier" from common_utils here instead of defining LGBMClassifier with all parameters

Reply:
There was an error training the model using create_lightgbm_classifier() with the dummy data. Which is why just using an unyrained lightgbm model for the test.